.eslintrc

```diff
@@ -0,0 +1,14 @@
+{
+    "extends": ["eslint:recommended", "airbnb-base/legacy"],
```

Comment:
This style looks good to me except for the spaces in the arrays which I'm kinda attached to. 
Can we get them back ?